index.html

```diff
@@ -508,19 +516,12 @@ <h3>
           <li>If this object's {{[[Released]]}} internal slot is `true`, return
           <a>a promise resolved with</a> `undefined`.
           </li>
-          <li>Otherwise, let |promise:Promise| be <a>a new promise</a>.
-          </li>
-          <li>Run the following steps <a>in parallel</a>:
-            <ol>
-              <li>Run <a>release a wake lock</a> with |lock:WakeLockSentinel|
-              set to this object and |type:WakeLockType| set to the value of
-              this object's {{WakeLockSentinel/type}} attribute.
-              </li>
-              <li>Resolve |promise|.
-              </li>
-            </ol>
+          <li>Otherwise, run <a>release a wake lock</a> with
+          |lock:WakeLockSentinel| set to this object and |type:WakeLockType|
```

Comment:
not sure I understand "set to this  object"

Reply:
Note that "this object" is the language we've already been using here and in other steps. If just saying `this` is more idiomatic, I can change all occurrences in a separate PR.

Reply:
Só why not use backtick ` around 'this'

Reply:
The short answer is "because nobody had ever brought it up before" :-) I'm just following the language we had already been using -- this excerpt was reindented but otherwise left completely unmodified. I can change that, but I'd rather do it in a separate PR because we say that in other places too.

Reply:
Marking this as resolved, this change was landed in #304